core/src/main/java/org/kiwix/kiwixmobile/core/webserver/ZimHostActivity.java

```diff
@@ -0,0 +1,316 @@
+/*
```

Comment:
These zimHost files have been converted to kotlin, have you merged develop to your branch?

Reply:
I don't see a webserver directory.  Has it been moved?

Reply:
it is in the `app` module